blog/2024-06-03-ratify-1.2.0.mdx

```diff
@@ -0,0 +1,68 @@
+
+Read on to discover the exciting new features and how they can benefit you!
+
+## Key Features, Contributions, and Decprecations
+
+Key features in this release include:
+- Cosign Verifier Enhancements 
+- Kubernetes Multi-Tenancy Support
+- Key Management Provider
+
+Contributions from the community:
+- [@duffney](https://github.com/duffney) made their first contribution: [support plugin logging #1254](https://github.com/deislabs/ratify/pull/1254)
+- [@mannbiher](https://github.com/mannbiher) made their first contribution: [fix: Set IdleTimeout for http.Server #1418](https://github.com/deislabs/ratify/pull/1418)
+
+Deprecations:
+- `CertificateStore` is deprecated in favor of KeyManagementProvider. Please migrate to KeyManagementProvider by following guide here. Support will be removed in Ratify v2.0.0
+
+## A brand new Cosign experience! 
+
+Starting in v1.2.0, Ratify now has more comprehensive Cosign verification capabilities. 
+
+Verification of Cosign signatures is a critical validation. As such, the Cosign verifier is now a built-in verifier along with the Notation verifier. Beyond first class support, the Cosign verifier introduces a `Trust Policy` which binds a particular set of verification settings to a certain registry scope. Now, it's possible to specify multiple trust policies based on your registry, namespace, and/or image. 
+
+Cosign verification also includes a robust key experience that integrates directly with KMP resources. Expanded capabilities include: multiple key support from KMP or filesystem and RSA + elliptical keys across multiple sizes.
+
+<!-- TODO: update links -->
+
+Watch this short demo to see the Cosign Verifier in action. Or check out the [documentation](https://ratify.dev/docs/reference/verifiers/cosign) for more information.
+
+[![asciicast](https://asciinema.org/a/658139.svg)](https://asciinema.org/a/658139)
+
+## Support for Multi-Tenancy in Kubernetes
+
+Ratify now includes multi-tenancy support. This new feature is designed to improve resource efficiency within Kubernetes clusters by enabling both multi-team and multi-customer architectures. 
+
+Multi-tenancy primarily focuses on a multi-team model, allowing organizations to effectively manage namespaces and allocate resources. Key user roles, such as cluster admins, team admins, and team developers, now have defined access and management capabilities.
+
+The addition of multi-tenancy support aims to facilitate seamless deployment, robust security, and efficient resource utilization, ensuring a smoother and more scalable experience for all.
+<!-- TODO: update links -->
+Watch this short demo to see multi-tenancy in action. Or check out the [documentation](https://ratify.dev/docs/reference/multi-tenancy) for more information.
+
+[![asciicast](https://asciinema.org/a/PO8RPcuZ2qyShif80c55WQipj.svg)](https://asciinema.org/a/PO8RPcuZ2qyShif80c55WQipj)
```

Comment:
I'm not actually sure this is a demo video? @binbin-li's guide has a video per step I believe.

Reply:
this is the video for cluster admin and team admin setting up cluster-wide and namespaced resources. If we just need one demo, this one is appropriate as we also recommend users to try the whole workflow in the given doc.